i18n/zh/docusaurus-plugin-content-blog/2022-09-06-karmada-v1.3/karmada-v1.3.md

```diff
@@ -45,13 +47,13 @@ metadata:
   spec:
     targetCluster:
       clusterNames:
-      - member1
-      - member2
+        - member1
+        - member2
```

Comment:
Why format the yaml in this way?
Is there a guideline we can follow?

Reply:
It is just a set of goland `actions on save`:

![image](https://user-images.githubusercontent.com/78244870/223674914-e4f72a7e-1999-4e41-956c-93c4e6e8c2cf.png)

It's just a markdown format setting. It's not a guideline.

Reply:
> It's just a markdown format setting. It's not a guideline.

That's exactly what I'm worried about. Since no guidelines, what if another guy uses another plugin and tries to change it back?

Reply:
I tried turning the setting off and saving the file again. The previous changes didn't roll back, this means that the formatted file is also a reasonable writing.